src/entity.js

```diff
@@ -175,7 +176,7 @@ export class Entity {
 
     return this.getTransport()
       .update(this.getResource(), this.getId(), requestBody)
-      .then((updated) => response = updated)
+      .then(updated => { response = updated; })
```

Comment:
that.s contradictory. either we use the rule and then we need curlies(+semicolons) or we allow returning assignments. unfortunatelly they don.t have an exception for fat arrows (yet)